docs/UserGuide.md

```diff
@@ -156,9 +156,9 @@ Examples:
 
 ### Finding a person: `find`
 
-Format: ‘find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tags’’
+Format: `find n/NAME p/PHONE_NUMBER r/ROOM_NUMBER t/Tag`
 
-* any possible orders and combinations of the 3 parameters name, phone number and room number are applicable
+* any possible orders and combinations of the 4 parameters NAME, PHONE_NUMBER, ROOM_NUMBER, and TAG are applicable
 
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
```

Comment:
Currently the search isn't  case insensitive for tags, should we write that into the UG or fix it for the tags?